OpenSim/Simulation/Model/Model.h

```diff
@@ -943,6 +943,14 @@ OpenSim_OBJECT_NONTEMPLATE_DEFS(Model, ModelComponent);
     std::vector<SimTK::ReferencePtr<const OpenSim::Coordinate>>
         getCoordinatesInMultibodyTreeOrder() const;
 
+    /** A variant of getCoordinatesInMultibodyTreeOrder that returns names for Scripting users */
+    SimTK::Array_<std::string> getCoordinateNamesInMultibodyTreeOrder() {
+        SimTK::Array_<std::string> namesArray;
+        auto coords = getCoordinatesInMultibodyTreeOrder();
```

Comment:
`coords` is a `std::vector<T>`, so this can be rewritten as:

```c++
const auto coords = getCoordinatesInMultibodyTreeOrder();
SimTK::Array_<std::string> namesArray;
namesArray.reserve(coords.size());  // prevent a reallocation
for (const auto& coord : coords) {
    namesArray.push_back(coord.getName());
}